site/src/pages/RoadmapPage/ImportZot4PlanPopup.tsx

```diff
@@ -0,0 +1,157 @@
+      // Success; hide the modal
+      setShowModal(false);
+    } finally {
+      setBusy(false);
+    }
+  };
+
+  return (
+    <>
+      <Modal show={showModal} onHide={() => setShowModal(false)} centered className="ppc-modal transcript-form">
+        <Modal.Header closeButton>
+          <h2>Import Schedule from Zot4Plan</h2>
+        </Modal.Header>
+        <Modal.Body>
+          <Form className="ppc-modal-form">
+            <Form.Group>
+              <p>
+                If you use{' '}
+                <a target="_blank" href="https://zot4plan.com/" rel="noreferrer">
+                  Zot4Plan
+                </a>
+                , you can add all your classes from that schedule to a new roadmap in PeterPortal. Your schedule in
+                Zot4Plan and your current roadmaps will not be modified.
+              </p>
+              <p>Please enter the exact name that you use to save and load your Zot4Plan schedule, as shown here:</p>
+              <img
+                className="w-100"
+                src={helpImage}
+                alt="Screenshot of Zot4Plan's save feature where the schedule name is typically used"
+              />
+            </Form.Group>
+            <Form.Group controlId="ScheduleName">
+              <Form.Label className="ppc-modal-form-label">Schedule Name</Form.Label>
+              <Form.Control
+                type="text"
+                placeholder="Exact Zot4Plan schedule name"
+                onChange={(e) => setScheduleName(e.target.value)}
+              />
+              {scheduleName.length > 0 && scheduleName.length < 8 && (
+                <span className="import-schedule-warning">
+                  <ExclamationTriangle className="import-schedule-warning-icon" />
+                  No Zot4Plan schedule name contains less than 8 characters
+                </span>
+              )}
+            </Form.Group>
+            <Form.Group controlId="CurrentYear">
+              <Form.Label className="ppc-modal-form-label">Current Year</Form.Label>
+              <Form.Control as="select" onChange={(ev) => setStudentYear(ev.target.value)} value={studentYear}>
+                <option value="1" selected>
+                  I'm a first year
+                </option>
+                <option value="2">I'm a second year</option>
+                <option value="3">I'm a third year</option>
+                <option value="4">I'm a fourth year</option>
+              </Form.Control>
+            </Form.Group>
+          </Form>
+          <Button variant="primary" disabled={busy || scheduleName.length < 8} onClick={handleImport}>
+            {busy ? 'Importing...' : 'Import'}
+          </Button>
+        </Modal.Body>
+      </Modal>
+      <Button
+        variant={darkMode ? 'dark' : 'light'}
+        className="ppc-btn import-schedule-btn"
+        onClick={() => setShowModal(true)}
+      >
+        <FileEarmarkArrowDown className="import-schedule-icon" />
```

Comment:
A file icon doesn't really make sense, I think something like the arrow going right into a square (similar to log in) or cloud with down arrow would make more sense